theano/sandbox/cuda/basic_ops.py

```diff
@@ -3047,6 +3049,7 @@ def c_code(self, node, name, inputs, outputs, sub):
         out = outputs[0]
         fail = sub['fail']
         inplace = int(self.inplace)
+        set_instead_of_inc = int(self.set_instead_of_inc)
```

Comment:
I don't fins tests for this set_subtensor on the gpu. Could you add a test in the file sandbox/cuda/tests/test_basic_ops.py that test this?

Otherwise, this PR look great.
thanks.